static/css/cf-enhancements.less

```diff
@@ -804,3 +807,38 @@ tbody {
     name: EOF
     eof: true
 */
+
+
+/* topdoc
+  name: Warning
+  family: cf-forms
+  notes:
+    - "Adds Warning state to form fields"
+  tags:
+    - cf-forms
+*/
+
+@input-warning:     @gold;
+
+input[type="text"],
```

Comment:
In the case of attribute selectors, I'd suggest using double quotes, to mirror the HTML standard.